src/main/java/org/ecommerce/Main.java

```diff
@@ -1,24 +1,49 @@
 package org.ecommerce;
 
-import org.ecommerce.controllers.CustomerController;
 import org.ecommerce.controllers.UserController;
-import org.ecommerce.repositories.CustomerRepository;
-import org.ecommerce.services.CustomerService;
-import org.ecommerce.services.PasswordServiceImpl;
+import org.ecommerce.repositories.UserRepository;
+import org.ecommerce.services.EmailService;
+import org.ecommerce.services.PasswordService;
+import org.ecommerce.services.impl.UserServiceImpl;
+import org.ecommerce.services.impl.EmailServiceImpl;
+import org.ecommerce.services.impl.PasswordServiceImpl;
 
 import java.util.HashMap;
 import java.util.Map;
 
 public class Main {
-    private static final Map<String, Object> REQUEST = new HashMap<>();
-    private static final CustomerRepository customerRepository = new CustomerRepository();
-    private static final UserController customerController = new CustomerController(new CustomerService(customerRepository), new PasswordServiceImpl(customerRepository));
+    private static final Map<String, String> request = new HashMap<>();
+    private static final UserRepository customerRepository = new UserRepository();
+    private static final PasswordService passwordService = new PasswordServiceImpl(customerRepository);
+    private static final EmailService emailService = new EmailServiceImpl();
+    private static final UserServiceImpl customerService = new UserServiceImpl(customerRepository, passwordService, emailService);
+
+    private static final UserController customerController = new UserController(customerService, passwordService);
+
     public static void main(String[] args) {
-        REQUEST.put("id", 1L);
-        REQUEST.put("passwords", Map.of("oldPassword", "Mario123!", "newPassword", "1357924680mM!"));
-        var response = customerController.changePassword(REQUEST);
+        request.put("id", "1"); // TODO: Shouldn't this id be generated by the db?
+        request.put("firstName", "Brian");
+        request.put("lastName", "Vega");
+        request.put("email", "BrianVega@test.com");
+        request.put("password", "@Password123");
+        request.put("phoneNumber", "341234567");
+        request.put("address", "Providencia #44");
+        request.put("preferences", "preference1, preference2, preference3");
+
+        var response = customerController.processCreationForm(request);
```

Comment:
Add TODO note. Instead of having maps as input, read strings and parse them to objects/maps. Create parser class